testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -373,6 +377,16 @@ def new_tenant_client(self, name, tenant):
         )
         time.sleep(45)
 
+    def new_tenant_docker_client(self, name, tenant):
+        if not self.MT_DOCKER_CLIENT_FILES[0] in self.docker_compose_files:
+            self.extra_files += self.MT_DOCKER_CLIENT_FILES
+        logger.info("creating docker client connected to tenant: " + tenant)
+        self._docker_compose_cmd(
+            "run -d --name=%s_%s mender-client" % (self.name, name),
+            env={"TENANT_TOKEN": "%s" % tenant},
+        )
+        time.sleep(5)
```

Comment:
I like this idea

Reply:
+1

Reply:
Does it support `docker-compose` though?

Reply:
I have been looking at the SDK over the weekend. It looks pretty nice and we should use it in `docker_manager.py`, but (answering @kacf question) it does not abstract `docker-compose`, so for `docker_compose_manager.py` it will get harder. It can be workaround it (filtering from the running containers according to the project name) but I don't think it will result in cleaner code.

Reply:
Unfortunately not.